source/Generate/Decoder.elm

```diff
@@ -69,6 +72,10 @@ renderArrayBody : String -> Type -> String
 renderArrayBody name type_ =
     "list " ++ (renderPropertyDecoder name "Item" type_)
 
+renderDictBody : String -> Type -> String
+renderDictBody name type_ =
+    "dict " ++ (renderPropertyDecoder name "Item" type_)
```

Comment:
I think another name than "Item", perhaps "Property", would reduce risk of confusion with arrays.